pkg/kubelet/rkt/rkt.go

```diff
@@ -51,8 +51,8 @@ import (
 )
 
 const (
-	acVersion             = "0.6.1"
-	rktMinimumVersion     = "0.8.1"
+	acVersion             = "0.7.1"
+	supportedRktVersion   = "0.9.0"
```

Comment:
Hmm....do we want to fail kubelet if rkt version is off? I think we should let user run with rkt 0.9.1 if they want to -- but we need a clear warning/event.